src/IBM.Watson.SpeechToText.v1/IBM.Watson.SpeechToText.v1.csproj

```diff
@@ -24,11 +24,90 @@
         <DebugSymbols>true</DebugSymbols>
     </PropertyGroup>
 
+    <ItemGroup>
+      <Compile Remove="Examples\**" />
+      <EmbeddedResource Remove="Examples\**" />
+      <None Remove="Examples\**" />
+    </ItemGroup>
+
     <ItemGroup>
         <PackageReference Include="IBM.Cloud.SDK.Core" Version="1.1.0" />
-        <PackageReference Include="IBM.Watson.Common" Version="4.6.0" />
+        <PackageReference Include="IBM.Watson.Common" Version="5.0.0" />
         <PackageReference Include="Newtonsoft.Json" Version="12.0.3" />
         <PackageReference Include="JsonSubTypes" Version="1.6.0" />
     </ItemGroup>
 
+    <ItemGroup>
+      <Compile Remove="Test\Unit\SpeechToTextServiceGeneratedUnitTests.cs" />

```

Comment:
I think we can remove this itemgroup if we add tests above

Reply:
Fixed!